docs/new-idp-requirements.md

```diff
@@ -0,0 +1,26 @@
+# New IDP Requirements
+
+## Summary
+
+This document describes the minimum requirements for adding a new IDP (Identity Provider) to the Sigstore Public Good Deployment.
+
+## Requirements
+
+The key words "MUST", "MUST NOT", "SHOULD", "SHOULD NOT", and "MAY" in this document are
+to be interpreted as described in [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt).
+
+A new IDP must meet the following requirements:
+
+- MUST host a `/.well-known/openid-configuration` file.
+- MUST have a documented key rotation policy.
+- MUST have a documented signing key storage policy.
```

Comment:
I don't think we can require this for the same above reason, so let's have this be a may or should and the MUST is having a secured signing key